### DISCUSSION TITLE
弃用警告

### DISCUSSION THREAD

**Discussion by qvsijia** [ppoffice/hexo-theme-icarus#1094] [General]
hexo g后弹出弃用警告

Deprecation warning: use moment.updateLocale(localeName, config) to change an existing locale. moment.defineLocale(localeName, config) should only be used for creating a new locale See http://momentjs.com/guides/#/warnings/define-locale/ for more info.

**Comment by ppoffice** [comment]
@qvsijia 目前暂无更新moment.js的计划。

**Reply by ppoffice** [reply]
#1064
